lib/mutant/ast/regexp/transformer.rb

```diff
@@ -0,0 +1,82 @@
+module Mutant
+  module AST
+    module Regexp
+      class Transformer
+        include AbstractType
+
+        REGISTRY = {} # rubocop:disable MutableConstant
```

Comment:
@dkubb that is essentially what I've done with this `TypeRegistry`. It still needs to be a constant or a updatable at some class level